proto/rill/runtime/v1/catalog.proto

```diff
@@ -85,6 +85,7 @@ message MetricsView {
     string name = 1;
     string label = 2;
     string description = 3;
+    string property = 4;
```

Comment:
Consider `column` or `field` instead – I think those are more commonly used for table column names